pulsar-client-tools-test/src/test/java/org/apache/pulsar/client/cli/PulsarClientToolTest.java

```diff
@@ -186,7 +186,7 @@ public void testDurableSubscribe() throws Exception {
         Assert.assertFalse(future.isCompletedExceptionally());
         future.get();
         //wait for close
-        Awaitility.waitAtMost(Duration.ofDays(2000));
+        Awaitility.waitAtMost(Duration.ofMillis(2000));
```

Comment:
this statement is a no-op. it's the same as calling `Awaitility.await().atMost(Duration.ofMillis(2000))` . We usually omit `.atMost` since the 10 second default value is fine for most cases.

There's probably something missing from this change.

Reply:
Just to clarify, the condition is missing from the statement. Using Awaitility is a no-op unless `.until` or `.untilAsserted` is used.

Reply:
Thank you for your explanation.